src/CMakeLists.txt

```diff
@@ -15,15 +26,20 @@ target_compile_definitions(journald
         -DG_LOG_DOMAIN="eventd-journald")
 target_link_libraries(journald
     PRIVATE
-        ${EventdEvent_LIBRARIES}
+        ${Eventd_LIBRARIES}
         ${EventdPlugin_LIBRARIES}
+        ${EVENTDHELPERS_LIBRARY}
         ${SystemdJournal_LIBRARIES})
 set_target_properties(journald
     PROPERTIES
         PREFIX    ""
         NO_SONAME 1)
 
+pkg_config_variable(libeventd-plugin pluginsdir EVENTD_PLUGINS_DIR_INTERNAL)
+set(EVENTD_PLUGINS_DIR ${EVENTD_PLUGINS_DIR_INTERNAL}
```

Comment:
Oh, I understand it. The problem is that `CMAKE_INSTALL_PREFIX` is set completely independently here. If it isn't set right, the manpages go to a completely different prefix. Since this is a plugin (which, like I said, I don't think I've ever use separate `--prefix` settings for before), I think something like this should be fine:

```cmake
pkg_config_variable(libeventd prefix eventd_prefix)
if (NOT CMAKE_INSTALL_PREFIX STREQUAL eventd_prefix)
  message(FATAL_ERROR "CMAKE_INSTALL_PREFIX does not match eventd's prefix (${eventd_prefix}); this is not supported")
endif ()
```